docassemble/AssemblyLine/data/questions/al_saved_sessions.yml

```diff
@@ -1,267 +1,151 @@
-      if not next_id:
-          break
-  al_sessions_list = sessions_list_temp
-  del sessions_list_temp
----
-code: |
-  formatted_sessions_temp = []
-  for index, session in enumerate(al_sessions_list):
-    formatted_sessions_temp.append({
-      index: f"{session.get('title')}: {session.get('subtitle') if session.get('subtitle') else ''} ({ format_datetime(session.get('modtime'), format='MM/dd/yyyy hh:mm a') })"
-    })
-  formatted_sessions_temp.sort(key=lambda session: next(iter(session.values())))
-  al_formatted_sessions = formatted_sessions_temp
-  del formatted_sessions_temp
----
-code: |
-  # This is a list of some variables that should never
-  # get copied into a new session.
-  al_sessions_variables_to_remove = [
-    # Internal fields
-    '_internal',
-    'nav',
-    'url_args',
-    'device_local',
-    'allow_cron',
-    'feedback_form',
-    'github_repo_name',
-    'github_user',
-    'interview_short_title',
-    'metadata_title',
-    'multi_user',
-    'session_local',
-    'speak_text',
-    'user_local',
-    # Database-like fields we don't need to copy
-    'all_courts',
-    'macourts',
-    'court_emails',    
-    # AssemblyLine form-specific fields 
-    'al_form_type',
-    'al_version',    
-    'form_approved_for_email_filing',    
-    'interview_metadata',    
-    'package_name',
-    'package_version_number',
-    'user_has_saved_answers',        
-    # Variables that should be calculated fresh
-    'signature_date',
-    'al_court_bundle',
-    'al_user_bundle',
-    'case_name',
-    # Variables from saving/loading state
-    'al_formatted_sessions',
-    'al_sessions_copy_success',
-    'al_sessions_fast_forward_filtered_vars',
-    'al_sessions_fast_forward_session',
-    'al_sessions_filtered_vars',
-    'al_sessions_launch_new_session',
-    'al_sessions_list',
-    'al_sessions_new_session_id',
-    'al_sessions_preview_variables',
-    'al_sessions_save_session_snapshot',
-    'al_sessions_save_session_snapshot_success',
-    'al_sessions_snapshot_label',
-    'al_sessions_snapshot_results',
-    'al_sessions_source_session',
-    'al_sessions_variables_to_remove',
-    'al_simple_filtered_vars',
-    'filtered_vars_tmp',
-    'simple_filtered_vars_tmp',
-    'al_sessions_url_ask_snapshot',
-    'al_sessions_url_ask_fast_forward',
-    'AL_ORGANIZATION_HOMEPAGE',
-    'AL_ORGANIZATION_TITLE',
-    'Any',
-    'Dict',
-    'List',
-    'Optional',
-    'al_logo',
-    # Maybe we will want these to be optional? Depends on how good review screen is.
-    # 'docket_number',
-    # 'docket_numbers',
-    #'user_ask_role',
-    #'user_role',    
-    #'user_started_case',
-  ]
+      Give this answer set a name you can remember later
+continue button label: |
+  :save: Save answer set
+script: |
+  <script>
+  % if not user_logged_in():
+    $(".da-field-buttons .btn-primary").attr("disabled", true);
+    $(".da-form-group input").attr("disabled", true);
+  % endif
+  </script>  
```

Comment:
It will save the answer set still. It just won't be associated with a user profile so won't be discoverable. It's an anonymous session just like any other anonymous session. But maybe that should be blocked in the YAML logic.